### ISSUE TITLE
AmbiguousTimeError in get_clearsky_tamb

### ISSUE THREAD

**Issue by mdeceglie** [NREL/rdtools#61]
minimum code example:

```
import pandas as pd
import rdtools
times = pd.date_range('2012-11-12', '2014-01-01', freq='H', tz='America/New_York')
rdtools.get_clearsky_tamb(times, 40.8, 74.0)
```

traceback:
```
  File "<stdin>", line 1, in <module>
  File "/anaconda2/lib/python2.7/site-packages/rdtools/clearsky_temperature.py", line 37, in get_clearsky_tamb
    dt_daily = pd.date_range(times[0] - buffer, times[-1] + buffer, freq='D')
  File "/anaconda2/lib/python2.7/site-packages/pandas/core/indexes/datetimes.py", line 2042, in date_range
    closed=closed, **kwargs)
  File "/anaconda2/lib/python2.7/site-packages/pandas/util/_decorators.py", line 91, in wrapper
    return func(*args, **kwargs)
  File "/anaconda2/lib/python2.7/site-packages/pandas/core/indexes/datetimes.py", line 317, in __new__
    ambiguous=ambiguous)
  File "/anaconda2/lib/python2.7/site-packages/pandas/core/indexes/datetimes.py", line 526, in _generate
    ambiguous=ambiguous)
  File "pandas/_libs/tslib.pyx", line 4580, in pandas._libs.tslib.tz_localize_to_utc (pandas/_libs/tslib.c:77701)
pytz.exceptions.AmbiguousTimeError: Cannot infer dst time from Timestamp('2012-11-04 01:00:00'), try using the 'ambiguous' argument
```

proposed solution in line 37:
`  dt_daily = pd.date_range(times.date[0] - buffer, times.date[-1] + buffer, freq='D')`

**Comment by mdeceglie** [comment]
Thanks @amnag . Please go ahead and create a pull request to the **development** branch.

**Comment by amnag** [comment]
Hi @mdeceglie and @cdeline, I added notebooks to rdtools/docs that read in csv/pickle files to calculate degradation rates. These notebooks are up-to-date with the python code that @mdeceglie  used on the entire set on merged files. There are two versions, one is for NREL users and the other is public-facing. I added a couple of pickle files to the repo to enable the public notebooks to run, and also wanted to add a couple of csv files. But .gitignore prohibits addition of csv files. Is it OK to modify .gitignore to add a couple of csv files to the repo ?
